cedar-policy-validator/src/schema/action.rs

```diff
@@ -66,6 +66,16 @@ impl ValidatorActionId {
     pub fn context_type(&self) -> Type {
         self.context.clone()
     }
+
+    /// The `EntityType`s that can be the `principal` for this action.
+    pub fn applies_to_principals(&self) -> impl Iterator<Item = &EntityType> {
```

Comment:
IIRC, `EntityType` is a core type. I think the public API exposes `EntityTypeName` instead.

Reply:
I think this is just exposing it from the validator crate for internal use. If we want this to be a public API (which would be reasonable), we'd need to do a bit more work to expose the `ValidatorActionId` itself.

Reply:
Haha, just saw John's other comment -- this isn't the public API.

Reply:
I think this is what I want. `ValidatorSchema.action_entities()` returns `Entities` from `cedar-policy-core` so it's weird / confusing to return `EntityTypeName` from `cedar-policy` instead. I could see refactoring more validator APIs in a future PR, but since I'm adding right before the release I want these changes to be minimal.

Reply:
Technically no rush to get this in 3.2 though -- we've been allowing major & minor changes in internal crates in patches.